wagtail/contrib/simple_translation/forms.py

```diff
@@ -12,15 +12,33 @@ class CheckboxSelectMultipleWithDisabledOptions(forms.CheckboxSelectMultiple):
 
     def create_option(self, *args, **kwargs):
         option = super().create_option(*args, **kwargs)
+        # Disabling the option if it's in the disabled_values list
         if option["value"] in self.disabled_values:
             option["attrs"]["disabled"] = True
+        else:
+            # Note: only adding the target/action to the non-disabled ones so that
+            # disabled ones get ignored
```

Comment:
This was only in the demo code to explain, I think the comment is useful but let's condense.

```suggestion
            # Only set target/action if not disabled to ignore change on disabled items
```

Something like that?